### PR TITLE
Fix Sensor offset calibration #215

### PR DESCRIPTION
**Pull request by mobilegurudevelopment**
fix humidity calibration issue
correct humidity value in alertService

### CODE REVIEW

File: station/Classes/Core/MeasurementsService/Impl/MeasurementsServiceImpl.swift [hunk 1]
```diff
@@ -140,23 +140,24 @@ extension MeasurementsServiceImpl: MeasurementsService {
                 withOffset offset: Double,
```
Comment (rinat-enikeev) [inline station/Classes/Core/MeasurementsService/Impl/MeasurementsServiceImpl.swift:140]: is offset used in the method?

Comment (mobilegurudevelopment) [reply station/Classes/Core/MeasurementsService/Impl/MeasurementsServiceImpl.swift:140]: @rinat-enikeev no, It's old function

Comment (rinat-enikeev) [reply station/Classes/Core/MeasurementsService/Impl/MeasurementsServiceImpl.swift:140]: so why still in method arguments?

Comment (mobilegurudevelopment) [reply station/Classes/Core/MeasurementsService/Impl/MeasurementsServiceImpl.swift:140]: ok, let me remove it

Comment (mobilegurudevelopment) [reply station/Classes/Core/MeasurementsService/Impl/MeasurementsServiceImpl.swift]: @rinat-enikeev cleanup finished, please continue to review. Thanks

**Review by rinat-enikeev — APPROVED** [review]

**Pull request merged**